docs/rules/no-await-in-loop.md

```diff
@@ -0,0 +1,89 @@
+# Disallow accidental serial execution of parallelizable operations (no-await-in-loop)
```

Comment:
Could you also update this to reflect the new `meta.description` field of the rule?